package.json

```diff
@@ -11,7 +11,7 @@
     "typecheck": "flow",
     "prepublish": "npm run build",
     "test": "npm run lint && npm run typecheck && npm run test-src",
-    "test-src": "mocha 'src/**/__tests__/*.js'"
+    "test-src": "mocha \"src/**/__tests__/*.js\""
```

Comment:
Oh no. I did not know this doesn't work on Windows! I have this line in a lot of projects.